legged_ylo2_hw:
  loop_frequency: 300
  cycle_time_error_threshold: 0.002
  thread_priority: 95
  power_limit: 3
  contact_threshold: 40
